app/config/ConfigDecorator.scala

```diff
@@ -66,12 +60,11 @@ class ConfigDecorator @Inject()(
   lazy val preferencesFrontendService = decorateUrlForLocalDev(s"preferences-frontend").getOrElse("")
   lazy val contactHost = decorateUrlForLocalDev(s"contact-frontend.host").getOrElse("")
   lazy val citizenAuthHost = decorateUrlForLocalDev(s"citizen-auth.host").getOrElse("")
-  lazy val companyAuthHost = decorateUrlForLocalDev(s"company-auth.host").getOrElse("")
-  lazy val companyAuthFrontendHost = decorateUrlForLocalDev(s"company-auth-frontend.host").getOrElse("")
   lazy val taiHost = decorateUrlForLocalDev(s"tai-frontend.host").getOrElse("")
   lazy val formTrackingHost = decorateUrlForLocalDev(s"tracking-frontend.host").getOrElse("")
 
   lazy val identityVerificationHost = decorateUrlForLocalDev(s"identity-verification.host").getOrElse("")
+  lazy val identityVerificationPrefix = decorateUrlForLocalDev(s"identity-verification.prefix").getOrElse("mdtp")
```

Comment:
@vimalsuresh @anujakhare iv uplift should now go to iv-stubs for local and staging, real iv for everything else